code/game/objects/structures/tables_racks.dm

```diff
@@ -825,6 +825,11 @@
 	pass_flags_self = LETPASSTHROW //You can throw objects over this, despite it's density.
 	max_integrity = 20
 
+/obj/structure/rack/skeletal
+	name = "skeletal minibar"
```

Comment:
You sure you don't want to give it its own desc? It isn't called a rack, so it doesn't make much sense referencing the Middle Ages here.

Reply:
"Rattle me drinks!"?